aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -1248,6 +1249,7 @@ void gemm_and_bias(
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_TRANSA, transa);
   cublasOperation_t transb = transpose_mat2 ? CUBLAS_OP_T : CUBLAS_OP_N;
   computeDesc.setAttribute(CUBLASLT_MATMUL_DESC_TRANSB, transb);
+  computeDesc.setAttribute<int32_t>(CUBLASLT_MATMUL_DESC_SM_COUNT_TARGET, at::cuda::getCurrentDeviceProperties()->multiProcessorCount - at::globalContext().SMCarveout());
```

Comment:
Looks like these calls need ifdefs as apparently ROCM does not have the same API